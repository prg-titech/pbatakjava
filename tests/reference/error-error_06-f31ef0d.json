{
    "basename": "error-error_06-f31ef0d",
    "cmd": "{infile}",
    "infile": "tests/error_06/build.sh",
    "infile_hash": "72ac74f493fda557310ca63facdb3c4061892323e8acf813cc4f8fc8",
    "stdout": null,
    "stdout_hash": null,
    "stderr": "error-error_06-f31ef0d.stderr",
    "stderr_hash": "f2ff2bbe61bb5edc7e8ba1543a22aa6aef28a9b09c5977d2252b2b37",
    "returncode": 4
}