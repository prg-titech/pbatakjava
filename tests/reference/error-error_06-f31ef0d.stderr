tests/error_06/Line.java:1: error: no solution found in this component
